docs/zdgbook/GettingStarted.rst

```diff
@@ -149,19 +148,18 @@ inside `src` directory.
   $ touch src/poll/main/__init__.py
   $ touch src/poll/main/configure.zcml
 
-The last file we created is a configuration file called Zope
-Configuration Markup Language (ZCML). Soon we will add some boiler
-plate code inside ZCML file.
+The last file is a configuration file. The file ending stands for
+``Zope Configuration Markup Language``.
```

Comment:
Maybe keep the abbreviation `ZCML`.

Reply:
I think it is always good to know a little bit of the background, so I kept explaining what the abbr stands for.